compiler/src/iree/compiler/Codegen/Common/GPU/test/gpu_distribute_scf_for.mlir

```diff
@@ -41,16 +56,28 @@ func.func @distribute_to_y(%lb : index, %ub : index, %step: index, %output: memr
 
 // CHECK-LABEL: func.func @distribute_to_y
 //  CHECK-SAME: %[[LB:.+]]: index, %[[UB:.+]]: index, %[[STEP:.+]]: index
-//       CHECK:   %[[ID:.+]] = gpu.thread_id y
-//       CHECK:   %[[DIM:.+]] = gpu.block_dim y
+//   CHECK-DAG:   %[[ID:.+]] = gpu.thread_id y
+//   CHECK-DAG:   %[[DIM:.+]] = gpu.block_dim y
 //       CHECK:   %[[YLB:.+]] = affine.apply affine_map<()[s0, s1, s2] -> (s0 * s1 + s2)>()[%[[ID]], %[[STEP]], %[[LB]]]
 //       CHECK:   %[[YSTEP:.+]] = affine.apply affine_map<()[s0, s1] -> (s0 * s1)>()[%[[DIM]], %[[STEP]]]
 //       CHECK:   scf.for %[[IV:.+]] = %[[YLB]] to %[[UB]] step %[[YSTEP]] {
 //       CHECK:     memref.store %{{.+}}, %{{.+}}[%[[IV]]] : memref<?xf32>
 
+// NO-BLOCK-DIM-LABEL: func.func @distribute_to_y
```

Comment:
These tests are duplicated with the above in the sense they don't check additonal stuff? It's fine to just have one of them and remove the others--saves us efforts to update all of them later.